hooks/persistence-azure-monitor/hook/hook.js

```diff
@@ -0,0 +1,89 @@
+// SPDX-FileCopyrightText: 2022 iteratec GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// Fixed settings for the script
+const monitorApiVersion = '2016-04-01';
+const processingDate = new Date().toUTCString();
+
+async function handle({
+  getFindings,
+  scan,
+  workspaceId = process.env["MONITOR_WORKSPACE_ID"],
+  sharedKey = process.env["MONITOR_SHARED_KEY"],
+  logTypePrefix = process.env["MONITOR_LOGTYPE_PREFIX"],
+  crypto = require("crypto"),
+  fetch = require('node-fetch')
+}) {
+  if (!(workspaceId && sharedKey)) {
+    console.error("Missing Workspace ID or shared key. Please provide them in the MONITOR_WORKSPACE_ID and MONITOR_SHARED_KEY environment variables");
+    return 1;
```

Comment:
The return value does not do anything here as it isn't intepreted by the hook-sdk, see: https://github.com/secureCodeBox/secureCodeBox/blob/9d4e49a3f71fe96709bbf43b75e43aafdd8c556d/hook-sdk/nodejs/hook-wrapper.js#L154

You can exit directly using process.exit(1) or throwing an error